make_scripts_riscv/project.mk

```diff
@@ -187,8 +187,8 @@ all:
 ifeq ("$(OS)","Windows_NT")
 else
 ifeq ("$(CONFIG_CHIP_NAME)", "BL602")
-	#@cd $(BL60X_SDK_PATH)/image_conf; python3 flash_build.py $(PROJECT_NAME) $(CONFIG_CHIP_NAME)
-	@cd $(BL60X_SDK_PATH)/image_conf; ./flash_build $(PROJECT_NAME) $(CONFIG_CHIP_NAME)
+	@cd $(BL60X_SDK_PATH)/image_conf;python3 -m pip install -r requirements.txt; python3 flash_build.py $(PROJECT_NAME) $(CONFIG_CHIP_NAME)
```

Comment:
I don't think that we need on every flash to install already installed requirements. Can you made it so it will install them only if they are not installed yet?

Reply:
pip keeps track of installed deps, it won't actually install the packages on every run, it'll just check, and if all the deps version requirements are met, skip installing. 
I can add a condition to check if the requirements exist and install if they don't, but that will require calling into pip too and will complicate the build logic. What do you think?

Reply:
Sounds fair! Let me merge this